2023/spring/Scheffler-MW@0230PM/praise-ogwuche.md

```diff
@@ -0,0 +1,20 @@
+# Praise Ogwuche
+# Whimsical Bio
+
+**I am an M25**
+
+*Major: Computational Sciences and Natural Sciences*
+*Concentrations: Data Science and Cells and Organisms*
+
+
+***Short Bio***: 
+> My name is Praise and I love music
+> "Live Honestly!"
```

Comment:
In the preview mode, the two quote lines are appearing on the same line rather than on different lines.